EssentialsChat/src/main/java/com/earth2me/essentials/chat/EssentialsChatPlayerListenerNormal.java

```diff
@@ -63,13 +68,15 @@ public void onPlayerChat(final AsyncPlayerChatEvent event) {
                     if (event.getMessage().charAt(0) == ess.getSettings().getChatShout() || (event.getMessage().charAt(0) == ess.getSettings().getChatQuestion() && ess.getSettings().isChatQuestionEnabled())) {
                         event.setMessage(event.getMessage().substring(1));
                     }
-                    event.setFormat(tl(chatStore.getType() + "Format", event.getFormat()));
+                    event.setFormat(tl(chatStore.getType().name().toLowerCase(Locale.ROOT) + "Format", event.getFormat()));
                     event.getRecipients().removeIf(player -> !ess.getUser(player).isAuthorized("essentials.chat.receive." + chatStore.getType()));
-                    return;
+
+                    callChatEvent(event, chatStore.getType());
+                } else {
+                    user.sendMessage(tl("notAllowedTo" + StringUtils.capitalize(chatStore.getType().name().toLowerCase(Locale.ROOT))));
```

Comment:
No need to use StringUtils here
```suggestion
                    final String chatStoreName = chatStore.getType().name();
                    user.sendMessage(tl("notAllowedTo" + chatStoreName.substring(0, 1) + chatStoreName.substring(1).toLowerCase(Locale.ENGLISH)));
```